pom.xml

```diff
@@ -8,7 +8,7 @@
     <groupId>com.fasterxml.jackson.module</groupId>
     <artifactId>jackson-module-kotlin</artifactId>
     <name>jackson-module-kotlin</name>
-    <version>2.10.1-SNAPSHOT</version>
+    <version>2.10.2-SNAPSHOT</version>
```

Comment:
This is probably accidental, could you revert it? (2.10.1 not yet released)

Reply:
version is reverted

Reply:
version is reverted